tests/js/client/aql/aql-ternary.js

```diff
@@ -468,6 +491,35 @@ RETURN NOOPT(true) ?
       assertEqual(true, result[0]);
     },
 
+    testShortcutTernary : function () {
```

Comment:
I would like the same test with swapped positions.

`RETURN NOOPT(false) ? ASSERT(false, 'fail') : `

To make sure that also the "positive" code path is not taken if the condition yields FALSE